hazelcast/src/main/java/com/hazelcast/client/impl/StubAuthenticationException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.client.impl;
+
+import com.hazelcast.spi.annotation.PrivateApi;
+
+
+/**
+ * This is equivalent of AuthenticationException from client side. When this is thrown to client
+ * AuthenticationException comes out of from other side.
+ */
+@PrivateApi
```

Comment:
Since this class is in an `impl` package, the annotation as private API is redundant